### PR TITLE
Miscellaneous minor updates

### PR DESCRIPTION
**Pull request by ridgill**
The following minor updates:
- Update code assistant login flow
- Add terms to AI glossary ref page
- Move Jupyter notebook ref to watsonxai/100 page
- Spanish equivalent updates

### CODE REVIEW

File: content/Watsonx/WatsonxAI/100.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ toc: true
 ### [watsonx Assistant and Watson Discovery](https://techzone.ibm.com/collection/watson-enterprise/environments)
 
 
-## FAQs
+# FAQs
```
Comment (adams0619) [inline content/Watsonx/WatsonxAI/100.md:16]: suggestion: if you want the headings to show up in the TOC (table of contents) not he right hand then make this line an h2 _(or remove it)_ and also make the following line an h2...
https://github.com/ibm-build-lab/VAD-VAR-Workshop/blob/7ada6ea923977affb0eb3483f2f73638791290e0/content/Watsonx/WatsonxAI/100.md?plain=1#L20

**Review by adams0619 — COMMENTED** [review]
Left a comment or two about the TOCs in main FAQ page... but everything else looks good

File: content/Watsonx/WatsonxAI/100.md (hunk 1 above)
Comment (ridgill) [reply content/Watsonx/WatsonxAI/100.md]: I went ahead and removed it.  Good catch!

**Review by adams0619 — APPROVED** [review]
LGTM

**Pull request merged**